src/base/rss/rssarticle.h

```diff
@@ -49,18 +49,18 @@ namespace Rss
         Q_OBJECT
 
     public:
-        Article(Feed *parent, const QString &guid);
+        Article(Feed *parent);
 
         // Accessors
         bool hasAttachment() const;
-        const QString &guid() const;
+        const QString guid() const;
```

Comment:
ok, so should I just remove the `const` from the end of all the functions at rssarticle? Or just from the ones I changed from `QString &` to `QString`?